File: C/PRAK503-2210817220029-SitiAinurRahmawati.c
#include <stdio.h>
int maksimal (int, int);
int maksimal(int a, int b){
    if (a>b) {
        return a;}
    return b;}

int minimal (int, int);  
int minimal(int a, int b){
    if (a<b) {
        return a;}
    return b;}

int main(){
    int batas = 0;
    int maks = -100000;
    int minim = 100000;
    int bilangan;
    scanf("%d", &bilangan);
    while(batas < bilangan){
        int nilai;
        scanf("%d", &nilai);
        maks = maksimal(maks, nilai);
        minim = minimal(minim, nilai);
        batas++;}
    printf("%d %d",maks,minim);
}

File: PYTHON/PRAK503-2210817220029-SitiAinurRahmawati.py
def maksimal(a, b) :
    if a>b :
        return a
    return b

def minimal (a, b) :
    if a<b :
        return a
    return b

batas = 0
maks = -100000
minim = 100000
bilangan=int(input())

while batas < bilangan :
    nilai = map(int, input().split())
    for nilai in nilai :
        maks = maksimal(maks, nilai)
        minim = minimal(minim, nilai)
        batas=batas+1
    print(maks, minim)